_i18n/es/python_backend.md

```diff
@@ -0,0 +1,44 @@
+## Acerca de Coopdevs
+
+Coopdevs es una entidad que aporta software libre a la economía social i solidaria. Surgida desde profesionales del sector tecnológico de Barcelona con inquietudes sociales, proporciona amplios conocimientos técnicos i metodologías de trabajo en este ámbito económico. Entre muchos otros proyectos, impulsamos [Katuma](http://katuma.org/), formamos parte del equipo de desarrollo de [Open Food Network](https://openfoodnetwork.org/) y desarrollamos [TimeOverflow](https://www.timeoverflow.org/).
+
+## Acerca del trabajo
+
+Ahora buscamos gente con espíritu cooperativo que tenga ganas de compartir este reto aportando su experiencia de desarrollo de software en un momento importante para hacer crecer el proyecto.
+
+La candidata se unirá principalmente al equipo que trabaja en la linea de negocio de ERP para cooperativas que se desarrolla en Python. Esta unidad se encarga de acompañar y ayudar a cooperativas y otras entidades a la sistematización de sus procesos internos y su optimización con software ERP. Las tareas són:
+
+* Implementaciones de ERP en empresas
+* Integraciones entre ERP i servicios externos
+* Acompañamiento a otros miembros junior del equipo
+* Desarrollo de backend web
+
+Además, dada la rotación de responsabilidades y diversidad de proyectos que tenemos conlleva que el trabajo no se limite a este ámbito y que también se dediquen horas a otros proyectos como Katuma, TimeOverflow o proyectos específicos para otros clientes, siempre en el ámbito web.
+
+Por tanto, buscamos alguna persona que cumpla con los siguientes criterios:
+
+* Experiencia y bases sólidas en diseño y programación orientada a objetos
+* Haber desarrollado aplicaciones web con frameworks open source en Python o Ruby tales como Django, Flask, Ruby on Rails, Sinatra, etc.
+* Experiencia y uso habitual de tests unitarios y de integración
+* Conocimiento de Git suficiente para gestionar todos los aspectos de un repositorio
+* Haber practicado code review en proyectos anteriores
+* Nivel medio de inglés escrito y leído
+* Saber trabajar en equipo
+* Capacidad de comunicación
+
+Algunas de las tecnologías que se usan son: Postgresql, Python, Rails, Ansible, Odoo y Git.
+
+Aspectos que valoramos muy positivamente:
+
+* Haber contribuido en proyectos open source
+* Interés en mejorar el ecosistema de la economía social y solidaria y la participación en proyectos de este ámbito
+* Experiencia con algun ERP escrito en Python
+* Conocimientos de optimización de bases de datos relacionales
+* Conocimientos de frontend web: JavaScript, CSS y HTML
+* Familiaridad con la línea de comandos
+* Interés en Data-Drive: basar las decisiones en datos empíricos a partir de sistemas de business intelligence, analíticas y monitorización
+
+<br/>
```

Comment:
do you need this?

Reply:
I use it to separate this last sentence from the main text